src/swig_python/contrib/gdk_example_liquid_amp.py

```diff
@@ -187,14 +188,16 @@ def login_with_mnemonic(cls, mnemonic):
     def login_with_pin(cls, pin):
         self = cls()
         pin_data = open(self.PIN_DATA_FILENAME).read()
-        self.session = gdk.Session({'name': 'liquid'})
+        self.session = gdk.Session({'name': self.NETWORK_NAME})
         credentials = {'pin': str(pin), 'pin_data': json.loads(pin_data)}
         self.session.login_user({}, credentials).resolve()
         self.fetch_subaccount()
         return self
 
     """Do not use this to instantiate the object, use create_new_wallet or login_with_*"""
     def __init__(self):
+        self.NETWORK_NAME = 'testnet-liquid' 
```

Comment:
This should be a global really, with a comment to change it to 'liquid' for mainnet.

Reply:
Not sure to get what u mean here, my knowledge of Python is limited. Please, tell me if now it's OK

Reply:
Yes, looking good thanks